htdocs/comm/card.php

```diff
@@ -362,6 +372,18 @@
 		}
 		print '</td></tr>';
 
+		if (!empty($conf->accounting->enabled)) {
```

Comment:
Thanks to use the new syntax
 if isModEnabled('accountancy')

Reply:
Oups ! Thanks @eldy , I pushed a patch just now